spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/AbstractShellIntegrationTest.java

```diff
@@ -16,10 +16,10 @@
 
 package org.springframework.cloud.dataflow.shell;
 
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.rules.TestName;
+import org.junit.jupiter.api.AfterAll;
```

Comment:
**[UNRELATED]** I converted the app registry test to Junit5 and therefore had to do the same across the test inheritance hieararchy.